test/secp256k1_test.exs

```diff
@@ -0,0 +1,72 @@
+defmodule Bitcoinex.Secp256k1Test do
+  use ExUnit.Case
+  doctest Bitcoinex.Secp256k1
+
+  alias Bitcoinex.Secp256k1
+
+  @valid_signatures_for_public_key_recovery [
```

Comment:
Can you include more signature tests? Are there more comprehensive test vectors from libsecp256k1 that can be pulled in?

Reply:
Added additional tests.